DuckDuckGo/VPNAutoShortcuts.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  VPNAutoShortcuts.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import AppIntents
+import Foundation
+
+#if ALPHA || DEBUG
+@available(iOS 17.0, *)
+struct VPNAutoShortcutsiOS17: AppShortcutsProvider {
+
+    @AppShortcutsBuilder
+    static var appShortcuts: [AppShortcut] {
+        AppShortcut(intent: EnableVPNIntent(),
+                    phrases: [
+                        "Connect \(.applicationName) VPN",
```

Comment:
How should localization be handled for Siri Shortcuts?

Reply:
This [isn't exactly trivial](https://sowenjub.me/writes/localizing-app-shortcuts-with-app-intents/) so I'm opening [a follow-up task to look into this](https://app.asana.com/0/1206580121312550/1209026082513680/f).